core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -62,38 +59,12 @@ object TypelevelPlugin extends AutoPlugin {
         scala <- githubWorkflowScalaVersions.value.init
         java <- githubWorkflowJavaVersions.value.tail
       } yield MatrixExclude(Map("scala" -> scala, "java" -> java.render))
+    },
+    githubWorkflowBuild ~= { steps =>
+      WorkflowStep.Sbt(List("headerCheckAll", "scalafmtCheckAll", "scalafmtSbtCheck")) +: steps
```

Comment:
Can these steps take arguments, even if just as one string?  Some of the scalafix linters require an arg.

Reply:
That's a good question. Yes, it seems we are already using that in http4s:
https://github.com/http4s/http4s/blob/1baf3d4f5fb135098b6b857a488b36cd39ca4a56/build.sbt#L41